admin_app/src/app/dashboard/components/Overview.tsx

```diff
@@ -157,22 +158,22 @@ const Overview: React.FC<OverviewProps> = ({ timePeriod }) => {
       title: "Total Queries",
       value: query_stats.n_questions,
       percentageChange: query_stats.percentage_increase,
-      Icon: ForumIcon,
+      Icon: ChatBubbleOutlineIcon,
       period: timePeriod,
     });
 
-    // Total Escalated Queries
+    // "Total Unsatisfactory Responses"
     statCardData.push({
-      title: "Total Escalated Queries",
+      title: "Unsatisfactory Responses",
```

Comment:
Sorry if I missed the discussion but why not Downvoted Responses?

Reply:
fixed!